core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -903,6 +905,56 @@ portfolio.pending.mediationResult.popup.selfAccepted.lockTimeOver=You have accep
 portfolio.pending.mediationResult.popup.openArbitration=Reject and request arbitration
 portfolio.pending.mediationResult.popup.alreadyAccepted=You've already accepted
 
+portfolio.pending.failedTrade.taker.missingTakerFeeTx=The taker fee transaction is missing.\n\n\
+  Without that the trade cannot be completed. No funds have been locked up and no trade fee has been paid. \
+  You can move the trade to failed trades.
+portfolio.pending.failedTrade.maker.missingTakerFeeTx=The peer's taker fee transaction is missing.\n\n\
+  Without that the trade cannot be completed. No funds have been locked up. \
+  Your offer is still available for other traders so you have not lost the maker fee. \
+  You can move the trade to failed trades.
+portfolio.pending.failedTrade.missingDepositTx=The deposit transaction (the 2of2 Multisig transaction) is missing.\n\n\
+  Without that the trade cannot be completed. No funds have been locked up but your trade fee has been paid. \
+  You can request at https://github.com/bisq-network/support/issues for getting reimbursed the lost trade fee. \
+  You can move the trade to failed trades.
+portfolio.pending.failedTrade.buyer.existingDepositTxButMissingDelayedPayoutTx=The delayed payout transaction is missing.\n\n\
+  Without that arbitration cannot be opened. The funds have been locked up in the deposit transaction and you must not \
+  send the fiat or altcoin payment to the BTC seller. Open a mediation ticket by clicking cmd+o (or ctrl+o). \
+  The mediator should make a suggested payout so that the seller receives their trade amount back and both traders get \
+  back their security deposit. That way there is no security risk and only the trade fee is lost. \
+  You can request at https://github.com/bisq-network/support/issues for getting reimbursed the lost trade fee.
+portfolio.pending.failedTrade.seller.existingDepositTxButMissingDelayedPayoutTx=The delayed payout transaction is missing.\n\n\
+  Without that arbitration cannot be opened. The funds have been locked up in the deposit transaction. If the buyer is \
+  also missing the delayed payout transaction he is instructed to not send the payment but open a mediation ticket. \
+  You should open a mediation ticket as well by clicking cmd+o (or ctrl+o). \
+  In case that the buyer has not sent the payment the mediator should make a suggested payout so that you receive \
+  your trade amount back and both traders get back their security deposit. Otherwise the trade amount should go to the buyer. \
+  You can request at https://github.com/bisq-network/support/issues for getting reimbursed the lost trade fee.
```

Comment:
```suggestion
portfolio.pending.failedTrade.seller.existingDepositTxButMissingDelayedPayoutTx=The delayed payout transaction is missing \
  but funds have been locked in the deposit transaction.\n\n\
  If the buyer is also missing the delayed payout transaction, they will be instructed to NOT send the payment and open \
  a mediation ticket instead. You should also open a mediation ticket with Cmd/Ctrl+o. \n\n\
  If the buyer has not sent payment yet, the mediator should suggest that both peers each get back the full amount of \
  their security deposits (with seller receiving full trade amount back as well). \
  Otherwise the trade amount should go to the buyer. \n\n\
  You can request a reimbursement for lost trade fees here: https://github.com/bisq-network/support/issues.
```